exercises/isogram/example.rs

```diff
@@ -0,0 +1,11 @@
+pub fn check(word: &str) -> bool {
+
+    // Filter all non-Alphabetic character out and collect them in a new String
+    let normalized_string: String = word.to_lowercase().chars().filter(|c| c.is_alphabetic()).collect();
+
+    // Find the char element from back and front and compare the index
+    let closure = |x: char, word: &str| word.find(x).unwrap() == word.rfind(x).unwrap();
```

Comment:
this should be named meaningfully saying what the function does. It determines whether a letter is unique.